packages/runtime-dom/src/modules/style.ts

```diff
@@ -8,7 +8,9 @@ export function patchStyle(el: Element, prev: Style, next: Style) {
   if (!next) {
     el.removeAttribute('style')
   } else if (isString(next)) {
-    style.cssText = next
+    if (prev !== next) {
+      style.cssText = next
```

Comment:
can we have a test to cover this pls?

Reply:
I'm lazy on it....Yeah. I add test for it.